src/main/java/core/AbstractPlayer.java

```diff
@@ -24,6 +24,7 @@
     public final int getPlayerID() {
         return playerID;
     }
+    public void setPlayerID(int id) {playerID = id;}
```

Comment:
Is there a need for this again?

Reply:
Indeed not...it crept back by default. And I think I now have a way round the original problem. Will remove again.